### DISCUSSION TITLE
Domotizzazione telecomando rolling code

### DISCUSSION THREAD

**Discussion by Pink1881** [indomus/forum#2595] [GENERALE - Discussioni generiche]
Ciao a tutti, 
vorrei domotizzare il telecomando del garage, ho visto questa guida: https://indomus.it/progetti/integrare-in-domotica-qualsiasi-telecomando-anche-rolling-code-tramite-contatto-pulito/
Ho quindi comprato un 4CHPROR3 ma non riesco a capire come fare i collegamenti, nella guida non sono specificati e nel manuale del prodotto ci sono solo per lampadine e prese ma non è il mio caso.
Potete aiutarmi?
Non sono molto pratico di circuiti...

**Comment by indomus** [comment]
> Ciao a tutti, vorrei domotizzare il telecomando del garage, ho visto questa guida: https://indomus.it/progetti/integrare-in-domotica-qualsiasi-telecomando-anche-rolling-code-tramite-contatto-pulito/ Ho quindi comprato un 4CHPROR3 ma non riesco a capire come fare i collegamenti, nella guida non sono specificati e nel manuale del prodotto ci sono solo per lampadine e prese ma non è il mio caso. Potete aiutarmi? Non sono molto pratico di circuiti...

è molto più semplice di quel che credi. devi collegare le singole coppie di contatti come spiegato nell'articolo ai contatti NO/COM dei relè del Sonoff.

**Reply by Pink1881** [reply]
Grazie, molto chiaro e semplice.
Il Sonoff  invece come devo alimentarlo?
Immagino convenga usare la 220, vedo i collegamenti per fase e neutro ma che tipo di cavo dovrei usare? Non mi è chiaro quanto consumi.
Ed immagino che il telecomando debba conservare la sua batteria per funzionare, giusto? Non viene alimentato dal Sonoff.

**Reply by KC-inDomus** [reply]
per comodità puoi usare anche 5 volt DC. alimentarlo a 220 AC è una possibilità, l'assorbimento dovrebbe essere intorno al singolo Watt.
sì, il telcomando viene azionato dal sonoff, ma rimane operativo come sempre.

**Reply by Pink1881** [reply]
Ah ottimo, penso userò la 220 per praticità (prendere un alimentatore da 5V è un pezzo ulteriore, occupa spazio e mi servirebbe una presa aggiuntiva per alimentarlo a sua volta), se assorbe solo 1W dovrebbe andar bene qualsiasi cavo, giusto?
Scusate ma come dicevo sono inesperto ma l'argomento mi interessa molto e vorrei fare tutto per bene ed in sicurezza.

**Reply by KC-inDomus** [reply]
sì, ovviamente parla con un elettricista per sicurezza.

**Comment by Pink1881** [comment]
Sono riuscito e funziona perfettamente, grazie per l'aiuto :)
